Jenkinsfile

```diff
@@ -287,10 +287,12 @@ def get_system_tests_pipeline() {
             sh """cd system-tests/
             scl enable rh-python35 -- python -m pytest -s --junitxml=./SystemTestsOutput.xml .
             """
-            junit "system-tests/SystemTestsOutput.xml"
-            archiveArtifacts "system-tests/logs/*.log"
           }  // stage
         } finally {
+          stage("System tests: Archive") {
+            junit "system-tests/SystemTestsOutput.xml"
+            archiveArtifacts "system-tests/logs/*.log"
```

Comment:
The same is valid here. To make this step ignore missing files, you can change the line to `archiveArtifacts allowEmptyArchive: true, artifacts: 'system-tests/logs/*.log'`. Feel free to choose the alternative you consider more adequate here.